packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -1080,16 +1082,25 @@ class _PopupMenuButtonState<T> extends State<PopupMenuButton<T>> {
   @override
   Widget build(BuildContext context) {
     assert(debugCheckHasMaterialLocalizations(context));
-    return widget.child != null
-      ? InkWell(
+
+    if (widget.child != null)
+      return Tooltip(
+        message:
+        widget.tooltip ?? MaterialLocalizations.of(context).showMenuTooltip,
+        child: InkWell(
           onTap: widget.enabled ? showButtonMenu : null,
           child: widget.child,
-        )
-      : IconButton(
-          icon: widget.icon ?? _getIcon(Theme.of(context).platform),
-          padding: widget.padding,
-          tooltip: widget.tooltip ?? MaterialLocalizations.of(context).showMenuTooltip,
-          onPressed: widget.enabled ? showButtonMenu : null,
-        );
+        ),
+      );
+
+    // If there is no child, it does not matter if icon == null because
```

Comment:
I don't feel like this comment is necessary, since the code itself feels pretty self-explanatory to me